utils/utils.tsx

```diff
@@ -79,14 +90,14 @@ export function isLinux() {
     return navigator.platform.toUpperCase().indexOf('LINUX') >= 0;
 }
 
-export function createSafeId(prop) {
+export function createSafeId(prop: {props: {defaultMessage: string}} | string | null): string | undefined {
```

Comment:
Based on the previous definitions, do we actually need this to take null as an argument?